### ISSUE TITLE
TypeError: Client.__init__() missing 1 required positional argument: 'callback_api_version'

### ISSUE THREAD

**Issue by tsundvoll** [equinor/isar#514]
**Describe the bug**
Getting
`TypeError: Client.__init__() missing 1 required positional argument: 'callback_api_version'` at startup

Seems like the issue is origniating from an updated version of the paho mqtt library
```python
from paho.mqtt.client import Client
```
where the `Client` class now has an required attribute `callback_api_version`.

**To Reproduce**
Start isar locally with `python main.py` with isar v1.16.10

**Expected behavior**
ISAR should start as normal

**Screenshots**
```
Exception ignored in: <function Client.__del__ at 0x7f0187bb2ef0>
Traceback (most recent call last):
  File "/opt/venv/lib/python3.10/site-packages/paho/mqtt/client.py", line 874, in __del__
    self._reset_sockets()
  File "/opt/venv/lib/python3.10/site-packages/paho/mqtt/client.py", line 1133, in _reset_sockets
    self._sock_close()
  File "/opt/venv/lib/python3.10/site-packages/paho/mqtt/client.py", line 1119, in _sock_close
    if not self._sock:
AttributeError: 'Client' object has no attribute '_sock'
Using environment variable for AZURE_CLIENT_ID
Traceback (most recent call last):
  File "/app/main.py", line 53, in <module>
    mqtt_client: MqttClient = MqttClient(mqtt_queue=queues.mqtt_queue)
  File "/opt/venv/lib/python3.10/site-packages/isar/services/service_connections/mqtt/mqtt_client.py", line 56, in __init__
    self.client: Client = Client()
TypeError: Client.__init__() missing 1 required positional argument: 'callback_api_version'
```

Old version:
![image](https://github.com/equinor/isar/assets/35451859/1ebf4325-a8c2-44ff-8eef-be5997a6f4e5)

New version:
![image](https://github.com/equinor/isar/assets/35451859/184c7f30-4796-41e0-ad27-67142c44824c)

**Comment by tsundvoll** [comment]
Adressed in #515

**Comment by tsundvoll** [comment]
Solved by #515 , closing issue
